test/runtime/Policies.go

```diff
@@ -1393,3 +1394,101 @@ var _ = Describe("RuntimeValidatedPolicyImportTests", func() {
 		Expect(res.Output().String()).Should(ContainSubstring(allowedVerdict), "Policy trace did not contain %s", allowedVerdict)
 	})
 })
+
+var _ = Describe("RuntimeValidatedPolicyDropAllTests", func() {
+	var once sync.Once
+	var logger *logrus.Entry
+	var vm *helpers.SSHMeta
+	var dropAll bool
+
+	initialize := func() {
+		logger = log.WithFields(logrus.Fields{"test": "RuntimeValidatedPolicyDropAllTests"})
+		logger.Info("Starting")
+		vm = helpers.CreateNewRuntimeHelper(helpers.Runtime, logger)
+		areEndpointsReady := vm.WaitEndpointsReady()
+		Expect(areEndpointsReady).Should(BeTrue())
+	}
+
+	BeforeEach(func() {
+		once.Do(initialize)
+		vm.PolicyDelAll()
+		res := vm.SetPolicyEnforcement(helpers.PolicyEnforcementDefault)
+		res.ExpectSuccess("Setting policyEnforcement to default")
+	})
+
+	AfterEach(func() {
+		vm.ValidateNoErrorsOnLogs(CurrentGinkgoTestDescription().Duration)
+		if CurrentGinkgoTestDescription().Failed {
+			vm.ReportFailed()
+		}
+	})
+
+	It("DROP_ALL Ingress Policy test", func() {
+		By("Starting cilium monitor in background to filter DROP requests")
+		ctx, cancel := context.WithCancel(context.Background())
+		res := vm.ExecContext(ctx, "cilium monitor --type drop")
+		defer cancel()
+
+		By("Create an endpoint with no labels to test DROP_ALL")
+		cmdStr := "docker run -dt --name ingressContainer --net=cilium-net tgraf/netperf"
+		vm.Exec(cmdStr).ExpectSuccess("Failed to create docker container")
+
+		endpointCmd := "cilium endpoint list | grep \"waiting-for-identity\" | awk '{ print $1}'"
```

Comment:
Avoid using `awk` and `grep`; it's brittle and is prone to break whenever the CLI output changes. Instead, do something like the following:

```
ingressContainer := "ingressContainer"
...
endpointIDS, err := vm.GetEndpointsIds()
Expect(err).To(BeNil(), "Unable to get IDs of endpoints")

ingressEpID, exists := endpointIDS[ingressContainer]
Expect(exists).To(BeTrue(), "Expected endpoint ID to exist for %s", ingressContainer)

ingressEpModel := vm.EndpointGet(ingressEpID)
Expect(ingressEpModel).To(Not(BeNil()), "nil model returned for endpoint %s", ingressEpID)
```

Then, you can access the field from the endpoint model directly. This will keep the test stable since the API has been declared stable. Do the same on line 1438.

Reply:
sure..will look into it